run/fillna_tier.py

```diff
@@ -167,82 +167,82 @@ def feature_engineering(score_file, tier_file):
-        patient.loc[pd.isna(patient['gnomadGeneZscore']), 'gnomadGeneZscore'] = patient['gnomadGeneZscore'].describe()['mean']
+        patient.loc[pd.isna(patient['gnomadGeneZscore']), 'gnomadGeneZscore'] = patient['gnomadGeneZscore'].describe()['min']
 
 
     patient.loc[patient['gnomadGenePLI'] == '-', 'gnomadGenePLI'] = np.NaN
     if np.all(pd.isna(patient['gnomadGenePLI'])):
         # Single variant query
-        patient.loc[pd.isna(patient['gnomadGenePLI']), 'gnomadGenePLI'] = feature_stat.loc['gnomadGenePLI', 'mean']
+        patient.loc[pd.isna(patient['gnomadGenePLI']), 'gnomadGenePLI'] = feature_stat.loc['gnomadGenePLI', 'min']
         patient['gnomadGenePLI'] = patient['gnomadGenePLI'].astype('float64')
     else:
         # Patient whole VCF
         patient['gnomadGenePLI'] = patient['gnomadGenePLI'].astype('float64')
-        patient.loc[pd.isna(patient['gnomadGenePLI']), 'gnomadGenePLI'] = patient['gnomadGenePLI'].describe()['mean']
+        patient.loc[pd.isna(patient['gnomadGenePLI']), 'gnomadGenePLI'] = patient['gnomadGenePLI'].describe()['min']
 
     
     patient.loc[patient['gnomadGeneOELof'] == '-', 'gnomadGeneOELof'] = np.NaN
     if np.all(pd.isna(patient['gnomadGeneOELof'])):
         # Single variant query
-        patient.loc[pd.isna(patient['gnomadGeneOELof']), 'gnomadGeneOELof'] = feature_stat.loc['gnomadGeneOELof', 'mean']
+        patient.loc[pd.isna(patient['gnomadGeneOELof']), 'gnomadGeneOELof'] = feature_stat.loc['gnomadGeneOELof', 'max']
         patient['gnomadGeneOELof'] = patient['gnomadGeneOELof'].astype('float64')
     else:
         # Patient whole VCF
         patient['gnomadGeneOELof'] = patient['gnomadGeneOELof'].astype('float64')
-        patient.loc[pd.isna(patient['gnomadGeneOELof']), 'gnomadGeneOELof'] = patient['gnomadGeneOELof'].describe()['mean']
+        patient.loc[pd.isna(patient['gnomadGeneOELof']), 'gnomadGeneOELof'] = patient['gnomadGeneOELof'].describe()['max']
     
     patient.loc[patient['gnomadGeneOELofUpper'] == '-', 'gnomadGeneOELofUpper'] = np.NaN
     if np.all(pd.isna(patient['gnomadGeneOELofUpper'])):
         # Single variant query
-        patient.loc[pd.isna(patient['gnomadGeneOELofUpper']), 'gnomadGeneOELofUpper'] = feature_stat.loc['gnomadGeneOELofUpper', 'mean']
+        patient.loc[pd.isna(patient['gnomadGeneOELofUpper']), 'gnomadGeneOELofUpper'] = feature_stat.loc['gnomadGeneOELofUpper', 'max']
         patient['gnomadGeneOELofUpper'] = patient['gnomadGeneOELofUpper'].astype('float64')
     else:
         # Patient whole VCF
         patient['gnomadGeneOELofUpper'] = patient['gnomadGeneOELofUpper'].astype('float64')
-        patient.loc[pd.isna(patient['gnomadGeneOELofUpper']), 'gnomadGeneOELofUpper'] = patient['gnomadGeneOELofUpper'].describe()['mean']
+        patient.loc[pd.isna(patient['gnomadGeneOELofUpper']), 'gnomadGeneOELofUpper'] = patient['gnomadGeneOELofUpper'].describe()['max']
```

Comment:
This one impute max value instead of min, is this intended?

Reply:
Yes, for these features, the lower the value, the more pathogenic it is.
Since we always impute SNP with the least pathogenic value, it should be max.

you can check such feature list at Line 626